examples/discovery-client/client.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+
+	"github.com/viam-modules/viamrtsp"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/robot/client"
+	"go.viam.com/utils/rpc"
+)
+
+func main() {
+	logger := logging.NewDebugLogger("client")
+	machine, err := client.New(
+		context.Background(),
+		"rawr-seanorg-main.nahz2tk7xm.viam.cloud",
+		logger,
+		client.WithDialOptions(rpc.WithEntityCredentials( 
+			"23c7c65f-7a77-401d-b551-b9288ec73b16",
+			rpc.Credentials{
+				Type:    rpc.CredentialsTypeAPIKey, 
+				Payload: "0tgr1rhxjltjpwxnvcwfalnn0ecpxr9y",
```

Comment:
lol i leaked my macbook machine api key

i removed it and replaced with new one